projects/msquic/Dockerfile

```diff
@@ -27,9 +27,6 @@ RUN apt-get update && \
 
 RUN git clone https://github.com/microsoft/msquic && \
     cd msquic && \
-    git submodule init submodules/clog && \
-    git submodule init submodules/openssl && \
-    git submodule init submodules/googletest && \
-    git submodule update
+    git submodule update --init && \
```

Comment:
Should you have the `&& \`? It didn't before. Maybe that's the source of your problem?

Reply:
shoot, yes.  Thank you